dashboard/test/models/pd/workshop_test.rb

```diff
@@ -275,7 +275,7 @@ class Pd::WorkshopTest < ActiveSupport::TestCase
     create(:pd_workshop_participant, workshop: workshop, enrolled: true, in_section: true, attended: true)
 
     assert workshop.account_required_for_attendance?
-    Pd::Enrollment.any_instance.expects(:send_exit_survey).times(2)
+    Pd::Enrollment.any_instance.expects(:send_exit_survey).times(1)
```

Comment:
For clarity, please rename the test case (say attended instead of in the section), explicitly create the first participant with `attended: false`, and remove the `in_section` parameters.